### ISSUE TITLE
Tarsal conjunctival location

### ISSUE THREAD

**Issue by daboe01** [obophenotype/human-phenotype-ontology#11555]
Preferred term label:
Tarsal conjunctival location

Synonyms

    Palpebral conjunctival location

    Tarsal location

    Conjunctiva, tarsal

Definition (free text, please give PubMed ID)
An anatomical spatial pattern or localization referring to the tarsal (palpebral) conjunctiva, which is the vascularized mucous membrane lining the inner posterior surface of the upper and lower eyelids. Used to modify ophthalmic phenotypes such as giant papillae or scarring. PMID:21307519

Parent term (use hpo.jax.org/app)
The anatomical localization of the specified phenotypic abnormality (HP:0012830)

Diseases characterized by this term ? (e.g. Orphanet or OMIM number)

    Ligneous conjunctivitis (OMIM:217090)

Your nano-attribution (ORCID)
0000-0001-9042-0508
